configs.go

```diff
@@ -2338,6 +2810,121 @@ func (config DeleteMyCommandsConfig) params() (Params, error) {
 }
 
+// SetMyNameConfig change the bot's name
+type SetMyNameConfig struct {
+	Name         string
+	LanguageCode string
+}
+
+func (config SetMyNameConfig) method() string {
+	return "setMyName"
+}
+
+func (config SetMyNameConfig) params() (Params, error) {
+	params := make(Params)
+
+	params.AddNonEmpty("name", config.Name)
+	params.AddNonEmpty("language_code", config.LanguageCode)
+
+	return params, nil
+}
+
+type GetMyNameConfig struct {
+	LanguageCode string
+}
+
+func (config GetMyNameConfig) method() string {
+	return "getMyName"
+}
+
+func (config GetMyNameConfig) params() (Params, error) {
+	params := make(Params)
+
+	params.AddNonEmpty("language_code", config.LanguageCode)
+
+	return params, nil
+}
+
+// GetMyDescriptionConfig get the current bot description for the given user language
+type GetMyDescriptionConfig struct {
+	LanguageCode string
+}
+
+func (config GetMyDescriptionConfig) method() string {
+	return "getMyDescription"
+}
+
+func (config GetMyDescriptionConfig) params() (Params, error) {
+	params := make(Params)
+
+	params.AddNonEmpty("language_code", config.LanguageCode)
+
+	return params, nil
+}
+
+// SetMyDescroptionConfig sets the bot's description, which is shown in the chat with the bot if the chat is empty
+type SetMyDescriptionConfig struct {
+	// Pass an empty string to remove the dedicated description for the given language.
+	Description string
+	//If empty, the description will be applied to all users for whose language there is no dedicated description.
+	LanguageCode string
+}
+
+func (config SetMyDescriptionConfig) method() string {
+	return "setMyDescription"
+}
+
+func (config SetMyDescriptionConfig) params() (Params, error) {
+	params := make(Params)
+
+	params.AddNonEmpty("description", config.Description)
+	params.AddNonEmpty("language_code", config.LanguageCode)
+
+	return params, nil
+}
+
+// GetMyShortDescriptionConfig get the current bot short description for the given user language
+type GetMyShortDescriptionConfig struct {
+	LanguageCode string
+}
+
+func (config GetMyShortDescriptionConfig) method() string {
+	return "getMyShortDescription"
+}
+
+func (config GetMyShortDescriptionConfig) params() (Params, error) {
+	params := make(Params)
+
+	params.AddNonEmpty("language_code", config.LanguageCode)
+
+	return params, nil
+}
+
+// SetMyDescroptionConfig sets the bot's short description, which is shown on the bot's profile page and is sent together with the link when users share the bot.
+type SetMyShortDescriptionConfig struct {
+	// New short description for the bot; 0-120 characters.
+	//
+	//Pass an empty string to remove the dedicated short description for the given language.
+	ShortDescription string
+	//A two-letter ISO 639-1 language code.
+	//
+	//If empty, the short description will be applied to all users for whose language there is no dedicated short description.
+	LanguageCode string
+}
+
+func (config SetMyShortDescriptionConfig) method() string {
+	return "setMyDescription"
```

Comment:
Wrong method for `SetMyShortDescriptionConfig`:
```suggestion
	return "setMyShortDescription"
```